Ghidra/Features/PDB/src/main/java/ghidra/app/util/bin/format/pdb/PdbKind.java

```diff
@@ -52,6 +52,25 @@ private static String toCamel(String name) {
 			}
 			if (makeUpper) {
 				c = Character.toUpperCase(c);
+				makeUpper = false;
+			}
+			buf.append(c);
+		}
+		return buf.toString();
+	}
+
+
+	private static String toSnake(String name) {
```

Comment:
If `toSnake()` is never called, there is no need to introduce it.

Reply:
Fixed!